GameDefinition/Content/ItemKind.hs

```diff
@@ -616,10 +617,10 @@ potion9 = potionTemplate
                : iaspects potionTemplate
   , ieffects = [ toOrganGood "more projecting" (40 + 1 `d` 10)
                , toOrganBad "pacified" (5 + 1 `d` 3)
-                   -- has to be weak, or would be too good when thrown at foes
+                   -- the malus has to be weak, or would be too good when thrown at foes
                , OnSmash (Explode "more projecting dew")
                , OnSmash (Explode "pacified mist") ]
-  -- , idesc    = ""
+  , idesc    = "Thick, sluggish fluid."
```

Comment:
The major effect is SkProject boost for many turns. That lets non-leader heroes shoot (normally they can only wait and be on lookout, brace or sleep), so the party has increased firepower. The momentary zeroed SkMelee is a balancing side-effect. The description underscores the disabled melee, perhaps even the focus, but it still confuses me wrt the ranged fighting ability boost.